### PR TITLE
[desktop] Add logout function

### PR DESCRIPTION
**Pull request by Thisak**
add logout for desktop version

### CODE REVIEW

**Comment by KumanoTanaka** [comment]
logic-updated

File: src/renderer/database/device.js [hunk 1]
```diff
@@ -52,6 +52,17 @@ export default {
       })
     })
   },
+  remove () {
```
Comment (KumanoTanaka) [inline src/renderer/database/device.js]: I'm sorry we have to drop whole database.
I changed the requirement document like below.
please change logic from removing device data to removing all data.
If you can `drop database` then it is the easiest way(I think).
logic-updated

**Comment by KumanoTanaka** [comment]
@Thisak 
please also update version in `package.json`.

File: src/renderer/database/device.js (hunk 1 above)
Comment (Thisak) [reply src/renderer/database/device.js]: Okay

**Comment by KumanoTanaka** [comment]
@Vuthisak 
I think you need to remove all the data related with user
not only user but also `balance`, `user_transaction`.
please check again.

**Comment by KumanoTanaka** [comment]
@Vuthisak 
I found that `nedb` dose not have feature that supports `drop db`.
but you can realize it by the following method.
see https://github.com/louischatriot/nedb/issues/84

**Comment by Thisak** [comment]
@KumanoTanaka  @m-toyoda I have updated code. Please review again.

**Comment by KumanoTanaka** [comment]
@Thisak 
LGTM
thank you for your support.

**Pull request merged**